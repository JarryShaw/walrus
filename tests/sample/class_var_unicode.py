class A:
    print(Ç := 666)
    print(Ç)
    print(Ç)
    print(Ç := 777)
    print(Ç)


print(A.__dict__.get('Ç'))
print(A.Ç)


class Ḑ:
    class Ȩ:
        print(Ģ := 888)
        print(Ģ)
        print(Ģ)
        print(Ģ := 999)
        print(Ģ)
    print(Ȩ.__dict__.get('Ģ'))
    print(Ȩ.Ģ)
