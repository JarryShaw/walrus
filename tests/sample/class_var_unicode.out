666
666
666
777
777
None
777
888
888
888
999
999
None
999
